ui/src/views/Authentication/SignIn/SignIn.tsx

```diff
@@ -106,38 +107,50 @@ const SignIn = (): JSX.Element => {
   const navigate = useNavigate();
   const showToast = useCustomToast();
 
+  const { mutateAsync } = useMutation({
+    mutationFn: (values: SignInPayload) => signIn(values),
+    mutationKey: ['signIn'],
+  });
```

Comment:
Why are we using mutation here?

Reply:
@sumitd94 mutation is used for POST requests in React Query

Reply:
Lol, I thought the internal query was handling it :P
nevermind, thanks